Library/DiscUtils.Xfs/BTreeExtentRoot.cs

```diff
@@ -80,16 +80,21 @@ public void LoadBtree(Context context)
                 if (Level == 1)
                 {
                     child = new BTreeExtentLeaf();
+                    if (context.SuperBlock.SbVersion == 5)
+                        child = new BTreeExtentLeafV5();
                 }
                 else
                 {
                     child = new BTreeExtentNode();
+                    if (context.SuperBlock.SbVersion == 5)
+                        child = new BTreeExtentNodeV5();
```

Comment:
Could you make it so that it's either one or the other? .. so we don't unecessarily create something for the GC to pick up.